dashboard/app/controllers/home_controller.rb

```diff
@@ -198,6 +198,13 @@ def terms_and_privacy
 
         @homepage_data[:censusQuestion] = school_stats.try(:has_high_school_grades?) ? "how_many_20_hours" : "how_many_10_hours"
         @homepage_data[:currentSchoolYear] = current_census_year
+        @homepage_data[:existingSchoolInfo] = {
+          id: teachers_school.id,
+          name: teachers_school.name,
+          country: 'US',
+          zip: teachers_school.zip,
+          type: teachers_school.school_type,
+        }
```

Comment:
yeah this is exactly where we need a new school info method. this will only return data for nces schools, not manually entered school info